docs/tutorial/upgrading-kubernetes.md

```diff
@@ -24,48 +27,14 @@ metadata:
   creationTimestamp: 2017-10-04T03:52:25Z
   name: simple.k8s.local
 spec:
-  api:
-    loadBalancer:
-      type: Public
-  authorization:
-    alwaysAllow: {}
-  channel: stable
-  cloudProvider: gce
-  configBase: gs://kubernetes-clusters/simple.k8s.local
-  etcdClusters:
-  - etcdMembers:
-    - instanceGroup: master-us-central1-a
-      name: a
-    name: main
-  - etcdMembers:
-    - instanceGroup: master-us-central1-a
-      name: a
-    name: events
-  iam:
-    legacy: false
-  kubernetesApiAccess:
-  - 0.0.0.0/0
-  kubernetesVersion: 1.7.2
-  masterInternalName: api.internal.simple.k8s.local
-  masterPublicName: api.simple.k8s.local
-  networking:
-    kubenet: {}
-  nonMasqueradeCIDR: 100.64.0.0/10
-  project: gce-project
-  sshAccess:
-  - 0.0.0.0/0
-  subnets:
-  - name: us-central1
-    region: us-central1
-    type: Public
-  topology:
-    dns:
-      type: Public
-    masters: public
-    nodes: public
+  ...
```

Comment:
I think this is a good edit of a long bit of text. These are nits, but I would keep the ellipses (...) to one line on either side of kubernetesVersion.